neuro_san/session/http_service_agent_session.py

```diff
@@ -86,6 +86,16 @@ def _get_request_path(self, function: str):
 
         return f"{scheme}://{self.use_host}:{self.use_port}/api/v1/{self.agent_name}/{function}"
 
+    def _get_headers(self) -> Dict[str, Any]:
+        """
+        Get headers for any outgoing request
+        """
+        headers: Dict[str, Any] = self.metadata
+        if headers is None:
+            headers = {}
+        headers["Connection"] = "close"
+        return headers
+
     def help_message(self, path: str) -> str:
         """
         Method returning general help message for http connectivity problems.
```

Comment:
In all API requests, make sure that our header includes "Connection: close" directive.